CHANGELOG.md

```diff
@@ -1,4 +1,19 @@
 ## 5.16.0 (Unreleased)
+  
+FEATURES:
+* **New Resource:** `google_clouddeploy_delivery_pipeline_iam_*` ([#6928](https://github.com/hashicorp/terraform-provider-google-beta/pull/6928))
+* **New Resource:** `google_compute_instance_group_membership` ([#6933](https://github.com/hashicorp/terraform-provider-google-beta/pull/6933))
+* **New Resource:** `google_discovery_engine_search_engine` ([#6919](https://github.com/hashicorp/terraform-provider-google-beta/pull/6919))
+* **New Resource:** `google_firebase_app_check_service_config` ([#6921](https://github.com/hashicorp/terraform-provider-google-beta/pull/6921))
+
+IMPROVEMENTS:
+* bigquery: promoted `table_replication_info` field on `resource_bigquery_table` resource to GA  ([#6929](https://github.com/hashicorp/terraform-provider-google-beta/pull/6929))
+* compute: added `confidential_instance_config.confidential_instance_type` field to `google_compute_instance`,  `google_compute_instance_template`, and `google_compute_region_instance_template` resources ([#6934](https://github.com/hashicorp/terraform-provider-google-beta/pull/6934))
+* networksecurity: remove unused custom code from `google_network_security_address_group` ([#6931](https://github.com/hashicorp/terraform-provider-google-beta/pull/6931))
+* provider: added an optional provider level label `goog-terraform-provisioned` to identify resources that were created by Terraform when viewing/editing these resources in other tools. ([#6924](https://github.com/hashicorp/terraform-provider-google-beta/pull/6924))
+
+BUG FIXES:
+* firebasehosting: fixed typing in `google_firebase_hosting_custom_domain` `issues.details` field ([#6926](https://github.com/hashicorp/terraform-provider-google-beta/pull/6926))
```

Comment:
This entry wasn't present in the raw output from changelog-gen when preparing changelogs for TPG - [see here](https://github.com/hashicorp/terraform-provider-google/pull/17240/commits/8afde64c3988ceb78bfdc833b4b10405b6349414).

Someone even [questioned why this wasn't in the recent TPGB release](https://github.com/GoogleCloudPlatform/magic-modules/pull/9902#issuecomment-1931666739). I suspect it may be that it was released but not present in the CHANGELOG - I'll investigate.

Reply:
I can't find a commit for that PR in the TPG codebase, and on the original MM PR you can only see one downstream PR that references it:

![Screenshot 2024-02-09 at 19 11 57](https://github.com/hashicorp/terraform-provider-google-beta/assets/15078782/ac31c427-093e-447b-ac5a-20b8c4d5eb51)

Reply:
Ah, the MM PR only generated diffs on the TPG repo, so there was only one PR to the downstream.

Reply:
This was a concern about nothing - the change is beta-only and [it is expected to be in this (v5.16.0) release](https://github.com/GoogleCloudPlatform/magic-modules/pull/9902#issuecomment-1936473575)